templates/esbuild.config.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path')
+const rails = require('esbuild-rails')
+
+require("esbuild").build({
```

Comment:
Nit but should we standardize around using single vs. double quotes here?

Reply:
[7f4109d](https://github.com/TheGnarCo/gnarails/pull/166/commits/7f4109d7618f197dff47d1aa146eb856853abad8)